crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -69,64 +69,64 @@ Keine Beschreibung gefunden.
 %s missbilligt den Gebrauch dieser Fähigkeit.
 %%%%
 You can't untransform!
-Du kannst dich nicht unverwandeln!
+Du kannst dich nicht zurückverwandeln!
 %%%%
 Sorry, you cannot become a bat while alive.
-Sorry, du kannst nicht eine Fledermaus werden wenn du lebendig bist.
+Entschuldige, du kannst keine Fledermaus werden, wenn du lebendig bist.
 %%%%
 You're already flying!
 Du fliegst schon!
 %%%%
 Sorry, you're not good enough to have a special ability.
-Sorry, du bist nicht gut genug, um eine besondere Fähigkeit zu haben.
+Entschuldige, du bist nicht gut genug, eine besondere Fähigkeit zu haben.
 %%%%
 "Use which ability? (? or * to list) "
-"Welche Fähigkeit verwenden? (? oder * aufzulisten) "
+"Welche Fähigkeit verwenden? (? oder * zum Auflisten) "
 %%%%
 You can't do that.
-Das kannst du nicht.
+Das kannst du nicht tun.
 %%%%
 You cannot do that while held in a net.
 Das kannst du nicht, solange du in einem Netz gefangen bist.
 %%%%
 You cannot do that while caught in a web.
-Das kannst du nicht, solange du in einem Gewebe gefangen bist.
+Das kannst du nicht, solange du in einem Spinnennetz gefangen bist.
```

Comment:
I was trying to avoid Spinnennetz because from v0.31 ribbon worms also create webs, and they're not spiders. Can we use Gespinst?